docs/nodes-and-miners/miner-mainnet.md

```diff
@@ -1,7 +1,7 @@
 ---
```

Comment:
I'm a little biased here, but the docs i created in stacksfoundation/miner-docs should be treated as the "source of truth" for this type of content. the content from the "prod" docs is a bit outdated. 

I'm not a fan of copy/pasting the content from there here, but i don't see a good alternative just yet

Reply:
Leaving this for now per our conversation, with the plan to copy/paste after initial fixes and then find a better solution.